dev-tools/scripts/releaseWizard.yaml

```
@@ -456,6 +456,16 @@ groups:
         comment: Make sure the edits done by `addVersion.py` are ok, then push
         cmd: git add -u .  && git commit -m "Add next minor version {{ next_version }}"  && git push
         logfile: commit-stable.log
+      - !Command
+        cmd: git checkout main
+        tee: true
+      - !Command
```

```suggestion
      - !Command
        cmd: git pull --ff-only
        tee: true
      - !Command
```

Maybe add a pull before committing?